whipper/test/test_common_mbngs.py

```diff
@@ -26,6 +26,24 @@ def testMissingReleaseDate(self):
 
         self.assertFalse(metadata.release)
 
+    def testTrackTitle(self):
+        """
+        Check if the "track title" is not "recording title"
+        see https://github.com/whipper-team/whipper/issues/192
+        """
+        # Using: The KLF - Space & Chill Out
+        # https://musicbrainz.org/release/c56ff16e-1d81-47de-926f-ba22891bd2bd
+        filename = 'whipper.release.c56ff16e-1d81-47de-926f-ba22891bd2bd.json'
+        path = os.path.join(os.path.dirname(__file__), filename)
+        handle = open(path, "rb")
```

Comment:
Even though the surrounding code is using an explicit `open()`, ..., `close()` construct, I think it's better to use a context manager here (as this isn't legacy code). The result would be something like this:

```python3
        with open(path, "rb") as handle:
            response = json.loads(handle.read().decode('utf-8'))
```

Reply:
I have updated this too. Thank you